src/proto/FunctionRpc.proto

```diff
@@ -254,6 +260,8 @@ message FunctionEnvironmentReloadResponse {
 
   // Status of the response
   StatusResult result = 3;
+
+  CapabilitiesUpdateStrategy capabilities_update_strategy = 4;
```

Comment:
What's the default strategy if capabilities_update_strategy is not specified by the worker?

Reply:
I was confused by this as well. Would be nice to have a comment in the code